notebooks/Beta_Visualization.ipynb

```diff
@@ -0,0 +1,729 @@
+{
```

Comment:
<p>Change the code in this cell:</p><ul><li>Delete class BetaManifold</li><li>Instead import class BetaDistributions, with a line such as: <code>from geomstats.information_geometry.beta import BetaDistributions</code></li><li>instantiate a manifold from this class and use its point_to_pdf method</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>